Formula/raylib.rb

```diff
@@ -0,0 +1,44 @@
+class Raylib < Formula
+  desc "Simple and easy-to-use library to learn videogames programming"
+  homepage "http://www.raylib.com"
+  url "https://github.com/raysan5/raylib/archive/1.9.1-dev.tar.gz"
+  version "1.9.1-dev"
+  sha256 "892357fb44d340eb7449c23c425d660d98b34b91434400e7610514ef02698600"
+
+  head "https://github.com/raysan5/raylib.git", :branch => "develop"
+
+  option "with-pic", "Compile static library as position-independent code"
+  option "without-static", "Don't Build a raylib static library"
+  option "without-shared", "Don't Build a raylib shared library"
+
+  depends_on "cmake" => :build
+
+  def install
+    args = *std_cmake_args
+    args << "-DWITH_PIC=ON" if build.with? "pic"
+    args << "-DSTATIC_RAYLIB=" + (build.without?("static") ? "OFF" : "ON")
+    args << "-DSHARED_RAYLIB=" + (build.without?("shared") ? "OFF" : "ON")
+    args << "-DMACOS_FATLIB=OFF" # User should specify --universal if they want fat libraries
+    args << "-DBUILD_EXAMPLES=OFF"
+    args << "-DBUILD_GAMES=OFF"
+
+    system "cmake", *args, "."
```

Comment:
. goes first, then all the inlined args, then *std_cmake_args

Reply:
Done.